internal/builders/generic/attest.go

```diff
@@ -59,7 +60,8 @@ run in the context of a Github Actions workflow.`,
 			// validated. This is done by CreateNewFileUnderCurrentDirectory.
 			if attPath == "" {
 				if len(parsedSubjects) == 1 {
-					attPath = fmt.Sprintf("%s.intoto.jsonl", parsedSubjects[0].Name)
+					filename := path.Base(parsedSubjects[0].Name)
```

Comment:
Adding a simple test in attest_test.go might also be good.

Reply:
done, thanks!